src/composables/useDark.js

```diff
@@ -0,0 +1,28 @@
+import { Theme } from '@/constants'
```

Comment:
You don't need to implement these composables yourself, you can just install the useDark package from @vueuse/core. Or am I missing something?